examples/experiments/ex5_ram_drg.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from artiq.coredevice.ad9910 import *
+from artiq.experiment import *
+
+from jax import (DRG, AD9910Manager, DRGType, JaxExperiment, RAMProfile,
+                 RAMType, SinaraEnvironment)
+
+# __all__ in an experiment module should typically only include the experiment class.
+# Specially, it cannot include the base experiment class.
+# ARTIQ discovers experiments by trying to load all objects that are subclasses of
+# artiq.experiment.Experiment. If __all__ includes the base experiment classs,
+# ARTIQ will try to load the base experiment class which results in an error.
+__all__ = ["RAMDRGExample"]
+
+
+class RAMDRGExample(JaxExperiment, SinaraEnvironment):
+    """Example experiment generating several DDS waveform, controlled by the AD9910 DRG and the RAM.
+
+    The following waveform are generated:
+    - Channel 0: DRG frequency sweep-up with a bidirectional amplitude RAM. The frequency increases
+        from 11 MHz to 36 Mhz, with 1 MHz increment every step; The ASF first sweeps-up from 0.1 to
+        1.0, then sweeps-down back to 0.1. RAM and DRG update at roughly the same time.
+        Note: See Data Latency (Pipeline Latency) Table in the Specification section of the AD9910
+        datasheet.
+    - Channel 1: Amplitude and frequency identical to Channel 0. Phase offset of the waveform
+        alternates from 0 to pi every step.
+    - Channel 2: Amplitude sweep-up then sweep down, identical to channel 0 and 1. A pi phase
+        offset is gradually injected into the waveform. 10 MHz in frequency.
+    - Channel 3: Phase sweep-up. The phase offset linearly increases from 0 to pi.
+    - Channel 4: Exact same waveform as Channel 1, but RAM and DRG updates are not aligned.
```

Comment:
Aren't there only 4 channels in the DDS?

Reply:
Yeah I messed that up, the shown channel 3 configuration isn't meant to be there.